task-queue/src/main/java/org/hawkular/metrics/tasks/impl/LeaseService.java

```diff
@@ -37,14 +37,18 @@
 
     public static final int DEFAULT_RENEWAL_RATE = 60;
 
+    @SuppressWarnings("unused")
```

Comment:
I'm not sure we should ignore those warnings. Ignoring a warning when a constructor/field is only used at runtime makes sense. But here the problem is that the task project is still a WIP.